corehq/apps/accounting/subscription_changes.py

```diff
@@ -243,7 +243,7 @@ def response_role_based_access(domain, new_plan_version):
             return True
         # temporarily disable this part of the downgrade until we
         # have a better user experience for notifying the downgraded user
-        # read_only_role = UserRole.get_read_only_role_by_domain(self.domain.name)
+        # read_only_role = get_read_only_role_for_domain(self.domain.name)
```

Comment:
How about just deleting this? It's been commented out since 2014: https://github.com/dimagi/commcare-hq/commit/bf14e6cce27b49e203796e260b58adf410b4eaed